lib/components/user/trip-notifications-pane.js

```diff
@@ -47,6 +47,7 @@ class TripNotificationsPane extends Component {
           </Radio>
 
           <SmallInfoText>
+            {/* TODO: populate either email or SMS below */}
             Note: you will be notified by [email|SMS]. This can be changed in your account settings once the trip has been saved.
           </SmallInfoText>
         </FormGroup>
```

Comment:
We should remove the federal holidays stuff below for now.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/174/commits/b4e99d7f6239a443ded667203754be15572ebf80